### PR TITLE
Create 2023-01-19-content.md

### PR DESCRIPTION
**Pull request by ptobia**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

N/A

## Select the type of change:

Adding CSE content release notes for 2023-01-19

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-cse/2023-01-19-content.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: January 13, 2023 - Content Release
```
Comment (kimsauce) [inline blog-cse/2023-01-19-content.md:2]: @ptobia Jan 19?

Comment (ptobia) [reply blog-cse/2023-01-19-content.md:2]: Yes, sorry for the late submission.

Comment (kimsauce) [reply blog-cse/2023-01-19-content.md:2]: no worries -- just to confirm, you want to keep it as Jan 13?

**Review by kimsauce — APPROVED** [review]
Approved - per @ptobia , keeping "Jan 13" in the title

**Pull request merged**